### PR TITLE
Frame fixes

### LINKED ISSUE #149: Change ShouldBeAbleToReconnectToADisconnectedBrowser implementation
The test is using `/frames/nested-frames.html` instead of `/frames/frame.html`

### PR DESCRIPTION
**Pull request by kblok**
closes #149

### CODE REVIEW

File: lib/PuppeteerSharp/Page.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ namespace PuppeteerSharp
     [DebuggerDisplay("Page {Url}")]
     public class Page : IDisposable
     {
-        public int DefaultNavigationTimeout { get; set; } = 30000;
+        public int DefaultNavigationTimeout { get; set; } = 30000000;
```
Comment (Meir017) [inline lib/PuppeteerSharp/Page.cs]: 30,000 seconds?

Comment (kblok) [reply lib/PuppeteerSharp/Page.cs]: oops :joy:

That was messing with my debugging. Good catch.

**Comment by kblok** [comment]
Ready to review @Meir017

File: lib/PuppeteerSharp.Tests/FrameUtils.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ public static string DumpFrames(PuppeteerSharp.Frame frame, string indentation =
             var result = indentation + Regex.Replace(frame.Url, @":\d{4}", ":<PORT>");
             foreach (var child in frame.ChildFrames)
             {
-                result += Environment.NewLine + DumpFrames(child, "    " + indentation);
+                result += "\n" + DumpFrames(child, "    " + indentation);
```
Comment (cdroulers) [inline lib/PuppeteerSharp.Tests/FrameUtils.cs]: Late to the party, but this makes the tests fail on Windows! Was NewLine not working on Mac or Linux? It should give "\n" on either platforms!

Comment (kblok) [reply lib/PuppeteerSharp.Tests/FrameUtils.cs]: The CI is a Windows Machine

Comment (Meir017) [reply lib/PuppeteerSharp.Tests/FrameUtils.cs]: maybe we should add CI on a linux machine as well?

Comment (cdroulers) [reply lib/PuppeteerSharp.Tests/FrameUtils.cs]: Looks like AppVeyor sets the line feed when checking out files to "\n" which is why it works on the CI.

https://help.appveyor.com/discussions/problems/1119-allow-changing-git-autocrlf-setting

I'll check out a working copy with the proper setting.

Comment (kblok) [reply lib/PuppeteerSharp.Tests/FrameUtils.cs]: Yeah. I've just got that in my local VM. We should fix this but in a more pragmatic way. I don't care if the browser returns a \n or a \r\n is not what we need to test.

I hate spending time making test happy.

**Pull request merged**